src/lexer.ml

```diff
@@ -662,7 +662,7 @@ let make_lexer_core keywords ghostKeywords startpos text reportRange inComment i
             Stream.Failure -> error "Bad string literal."
         in
         store c; string ()
-    | c when c < ' ' -> raise Stream.Failure

```

Comment:
Does this do the right thing when EOF occurs inside a string literal? EOF is represented in the lexer as a '\000' character.